spec/ysh-TODO-deprecate.test.sh

```diff
@@ -42,7 +42,7 @@ if ('foo42' ~ / <capture d+> /) {
 42
 ## END
 
-#### _status instead of _error.code
+#### _error.code (formerly _status)
```

Comment:
Let's remove this test case altogether

Because this is `spec/ysh-TODO-deprecate`

We have done the deprecation, so there's no more TODO!